Cannot compile package with gccgo

I am trying to compile hugo, getting this:
```
github.com/pkg/sftp
# github.com/pkg/sftp

code/go/src/github.com/pkg/sftp/server_statvfs_linux.go:17:35: error: reference to undefined field or method ‘X__val’
   Fsid:    uint64(uint64(stat.Fsid.X__val[1])<<32 | uint64(stat.Fsid.X__val[0])), // endianness?
                                   ^
code/go/src/github.com/pkg/sftp/server_statvfs_linux.go:17:69: error: reference to undefined field or method ‘X__val’
   Fsid:    uint64(uint64(stat.Fsid.X__val[1])<<32 | uint64(stat.Fsid.X__val[0])), // endianness?
```

--- a/server_statvfs_linux.go
+++ b/server_statvfs_linux.go
@@ -1,3 +1,5 @@
+// +build !gccgo,linux
+
 package sftp
 
 import (

--- a/server_statvfs_stubs.go
+++ b/server_statvfs_stubs.go
@@ -1,4 +1,4 @@
-// +build !darwin,!linux
+// +build !darwin,!linux gccgo
 
 package sftp
 

